internal/pkg/agent/application/coordinator/diagnostics_test.go

```diff
@@ -488,6 +500,7 @@ func newCoordinatorTestHelper(t *testing.T, agentInfo *info.AgentInfo, specs com
 
 	helper.coordinator = coordinator.New(
 		log,
+		configuration.DefaultConfiguration(),
```

Comment:
It would be better if the configuration was passed as a parameter to make it an explicit part of the test using the helper

Reply:
I updated this to use a created configuration, that adjusts the paths.